README.md

```diff
@@ -1,20 +1,21 @@
-# Kotatsu
```

Comment:
App`s name is not important in readme, yep~

Reply:
This is called styling if you don't like it never mind.